src/pages/groupComparison/GroupComparisonPage.tsx

```diff
@@ -507,4 +526,20 @@ export default class GroupComparisonPage extends React.Component<
             </PageLayout>
         );
     }
+    @action.bound
+    updateRankingChoices(
+        dropDownTitle: string,
+        isAlterationEnabled: number,
+        considerOnlyTCGAPanPathways: boolean,
+        isPercentageMatch: number,
+        selectedPathway: string
+    ) {
+        this.currentPathway = selectedPathway;
+        Object.assign(this.rankingChoices, {
+            dropDownTitle,
+            isPercentageMatch,
+            isAlterationEnabled,
+            considerOnlyTCGAPanPathways,
+        });
+    }
```

Comment:
We can also move this into the separate `GroupComparisonPathwayMapperUserSelectionStore` class.